### ISSUE TITLE
Libraries for only one python version in the wheel.

### ISSUE THREAD

**Issue by olivier65** [uNetworking/uWebSockets.py#7]
The wheel downloaded by pip has only one *.so library per OS in it: the one for Python 3.6; I had to compile and repackage uWebSockets.py myself to get a wheel with all three libs.

**Comment by alexhultman** [comment]
Yes but you don't really need the exact version 3.8 for your Python 3.8. It's perfectly fine running a 3.6 binary using a 3.8 Python interpreter, at least for the very limited APIs used. I have tested.

There's a central bug in every release prior to 3.8 though, and the python guys don't seem to backport fixes very actively. So when 3.9 releases, I plan on only supporting 3.8 and 3.9
